localization/english/event_localization/wc_draenor_events_l_english.yml

```diff
@@ -25,3 +25,15 @@
  WCDRA_25_OPT_A_very_positive:0 "One more chance for our brethren."
  WCDRA_25_OPT_A_positive:0 "Let them find home on Azeroth."
  WCDRA_25_OPT_A_negative:0 "Why these gates can't stay shut?"
+
+ WCDRA_200_TITLE:0 "Intruders Beyond the [ROOT.Char.Custom('GetPortalRift')]"
+ WCDRA_200_DESC_beginning:0 "Your soldiers report they faced armed forces coming out of the [ROOT.Char.Custom('GetDarkPortalRift')]. So far their numbers are unknown, but it's clear there is some movement beyond the veil.\n\nRumors say these aliens are"
+ WCDRA_200_DESC_draenei:0 "draenei survivors led by a powerful exarch blessed by the naaru."
```

Comment:
```suggestion
 WCDRA_200_DESC_draenei:0 "Draenei survivors led by a powerful Exarch blessed by the Naaru."
```

Reply:
Why you capitalized exarch? It's not a proper noun.
We don't capitalize races.

Reply:
Looked into exarch, some of the wikis choose to capitalize it. I used them as references. Will revert.
Didn't know about the latter.